app/build.gradle.kts

```diff
@@ -49,6 +49,8 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    namespace = "template"
```

Comment:
We'll need to tweak the setup build script to modify this line instead of the AndroidManifest now.